app/features/calculator.py

```diff
@@ -0,0 +1,892 @@
+        except (CalculationError, ValueError, TypeError) as e:
+            await ctx.send(f"An error occurred: {e!s}", ephemeral=True)
+
+    @calc.subcommand(sub_cmd_name="round", sub_cmd_description="Round a number to the nearest integer")
+    @interactions.slash_option(
+        name="x",
+        description="The number to round",
+        required=True,
+        opt_type=interactions.OptionType.STRING,
+    )
+    async def calc_round(self, ctx: interactions.SlashContext, x: str) -> None:
+        """Round a number to the nearest integer."""
+        try:
+            expression = f"round({x})"
+            result = evaluate_expression(expression)
+            await ctx.send(f"round({x}) = {result}")
+        except (CalculationError, ValueError, TypeError) as e:
+            await ctx.send(f"An error occurred: {e!s}", ephemeral=True)
+
+    @calc.subcommand(sub_cmd_name="ceil", sub_cmd_description="Round a number up to the nearest integer")
+    @interactions.slash_option(
+        name="x",
+        description="The number to round up",
+        required=True,
+        opt_type=interactions.OptionType.STRING,
+    )
+    async def calc_ceil(self, ctx: interactions.SlashContext, x: str) -> None:
+        """Round a number up to the nearest integer."""
+        try:
+            expression = f"ceil({x})"
+            result = evaluate_expression(expression)
+            await ctx.send(f"ceil({x}) = {result}")
+        except (CalculationError, ValueError, TypeError) as e:
+            await ctx.send(f"An error occurred: {e!s}", ephemeral=True)
+
+    @calc.subcommand(sub_cmd_name="floor", sub_cmd_description="Round a number down to the nearest integer")
+    @interactions.slash_option(
+        name="x",
+        description="The number to round down",
+        required=True,
+        opt_type=interactions.OptionType.STRING,
+    )
+    async def calc_floor(self, ctx: interactions.SlashContext, x: str) -> None:
+        """Round a number down to the nearest integer."""
+        try:
+            expression = f"floor({x})"
+            result = evaluate_expression(expression)
+            await ctx.send(f"floor({x}) = {result}")
+        except (CalculationError, ValueError, TypeError) as e:
+            await ctx.send(f"An error occurred: {e!s}", ephemeral=True)
+
+    @calc.subcommand(sub_cmd_name="rad", sub_cmd_description="Convert a degree value into radians")
+    @interactions.slash_option(
+        name="x",
+        description="Angle in degree",
+        required=True,
+        opt_type=interactions.OptionType.STRING,
+    )
+    async def calc_rad(self, ctx: interactions.SlashContext, x: str) -> None:
+        """Convert a degree value into radians."""
+        try:
+            expression = f"radians({x})"
+            result = evaluate_expression(expression)
+            result_pi = result
+            await ctx.send(f"{x}° in radians:\nDecimal: {result:.6f}\nIn terms of π: {result_pi}")
```

Comment:
result and result_pi only differ in decimal points, so why add it as a seperate result?